src/Tools/dotnet-watch/src/dotnet-watch.csproj

```diff
@@ -29,12 +29,6 @@
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
 
-   <ProjectReference
-      Include="$(RepoRoot)src\Framework\App.Runtime\src\Microsoft.AspNetCore.App.Runtime.csproj"
-      PrivateAssets="All"
-      ReferenceOutputAssembly="false"
-      SkipGetTargetFrameworkProperties="true" />
```

Comment:
Not useful due to `$(DoNotApplyWorkaroundsToMicrosoftAspNetCoreApp)` setting earlier in this file